api/Controllers/AuthController.cs

```diff
@@ -36,11 +38,32 @@ public IActionResult Login([FromBody]Login request)
             return Ok(token);
         }
 
-        private Account GetUserAccount(string email, string password)
+        [Route("registration")]
+        [HttpPost]
+        [Authorize]
```

Comment:
So any user can create an admin user?

Reply:
Roles gonna be added in next pull request